manifests/cf-manifest/scripts/generate-manifest.sh

```diff
@@ -17,7 +17,7 @@ if [ "${DISABLE_USER_CREATION}" = "false" ] ; then
 fi
 
 opsfile_args=""
-for i in ${PAAS_CF_DIR}/manifests/cf-manifest/operations.d/*.yml; do
+for i in "${PAAS_CF_DIR}"/manifests/cf-manifest/operations.d/*.yml; do
```

Comment:
I'd prefer `"${PAAS_CF_DIR}/manifests/cf-manifest/operations.d/*.yml"`, but I don't know how shellcheck would feel about that.

Reply:
Shellcheck was fine. But changing that as suggested changes the behaviour of the `for` loop...or more specifically the argument to the `for` loop. Shell interpolation gets in the way and the subsequent `opsfile_args` gets very much the wrong value.

Reply:
Good catch. I'd forgotten that chunks them all into one space-containing word.